netlify/plugins/airtable-feed/index.js

```diff
@@ -9,7 +9,23 @@ var dataDir;
 const fetchSubmissions = async() => {
     const response = await fetch(`https://api.airtable.com/v0/${AIRTABLE_BASE_ID}/Submissions?maxRecords=10000&view=Approved&api_key=${AIRTABLE_API_KEY}`);
     const data = await response.json();
-    return data.records;
+    return data.records.map(record=>{
+        record.fields.linktext=new URL(record.fields.URL).hostname;
+        const age = 2021-record.fields['Years unused'];
+        if (age < 1) {
+            record.fields.age = 'fresh';
+        }
+        else if (age == 1) {
+            record.fields.age = '1 year dusty';
+        }
+        else if (age > 1) {
```

Comment:
one guard that's not here is for an incorrect entry. if someone enters `5` for example, we'll end up with some huge numbers — I ended up putting a guard in for years less than `1990` to avoid having a "2000 years dusty" domain :joy:

Reply:
Definitely did the same thing same. What would you think we should do about that? Would you just have "super dusty" 😆 ?

Reply:
maybe leave it blank? this will be the case for e.g. `*.netlify.app` projects too